logger.js

```diff
@@ -1,14 +1,16 @@
 'use strict';
 // Log class
 
+var util      = require('util');
+var tty       = require('tty');
+
 var constants = require('haraka-constants');
+var logfmt    = require('logfmt');
```

Comment:
Couldn't we lazy load this?  Seems silly to require it for everyone whether they're using LOGFMT mode or not.

Reply:
We use this for the old key value comments too so all the old key=value stuff. The implementation is on like 275.